canonical-web-and-design/landscape-brochure.internal
# Add global nav and make it full width

## Done
Added the global nav

## QA
 - ./run
 - Go to any page
 - See that the global nav is present and full-width
